app/views/coc/divorce.scala.html

```diff
@@ -89,7 +82,7 @@ <h1 class="heading-xlarge">@Html(Messages("change.status.divorce.transferor.cond
                     <ul class="list-bullet space-smaller">
                         <li>@Html(Messages("change.status.divorce.transferor.conditional1.para3",TextGenerators.ukDateTransformer(cyEffectiveUntilDate, LanguageUtils.isWelsh(messages))))</li>
                         <li>@Html(Messages("change.status.divorce.transferor.conditional1.para4"))</li>
-                    </ul></br>
+                    </ul><br/>
```

Comment:
`section` and `subsection` AF classes are handy here, for vertical space

Reply:
resolved